### PR TITLE
[Feature] TicketValidation QrCode to Consent (EXPOSUREAPP-10194)

### PR DESCRIPTION
**Pull request by mtwalli**
WIP
### Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194

### CODE REVIEW

**Comment by vaubaehn** [comment]
@mlenkeit 
Will this be designed to be one-way? CWA DCC -> Vendor/Host
Or is a two-way design planned? CWA <-> Venodor/Host
Two-way would be nice to enable CWA to store and present tokens to gain access to events/locations etc.

**Comment by Ein-Tim** [comment]
Would be nice to get some information on what exactly is implemented here!

**Comment by mlenkeit** [comment]
> Would be nice to get some information on what exactly is implemented here!

@Ein-Tim  basically this: https://ec.europa.eu/health/sites/default/files/ehealth/docs/covid-certificate_traveller-onlinebooking_en.pdf (can be found via https://ec.europa.eu/health/ehealth/covid-19_en, search for _booking_)


> @mlenkeit Will this be designed to be one-way? CWA DCC -> Vendor/Host Or is a two-way design planned? CWA DCC -> Vendor/Host OTP -> CWA -> gate keeper Two-way would be nice to enable CWA to store and present tokens to gain access to events/locations etc.

@vaubaehn one way, if I understand your description correctly. Basically, it allows to securely share the result of the Business Rules check (aka _Gültigkeit prüfen_ feature) with a vendor.

**Comment by vaubaehn** [comment]
Hi @PhilippNowak96 ,
are you aware of the upcoming feature "DCC ticketing"? Is this something that could be interesting for Deutsche Bahn or is even already planned for implementation?

**Comment by vaubaehn** [comment]
Hi @mlenkeit 
> @vaubaehn one way, if I understand your description correctly. Basically, it allows to securely share the result of the Business Rules check (aka _Gültigkeit prüfen_ feature) with a vendor.

thanks for your response! Yes, you understood me correctly, even it was a bit cryptic...
I was thinking that it could be rather convenient if CWA was able to store tokens from vedors as QR codes for a check in process e.g., at airports, stadiums etc (that was what I meant to be a 2-way-design), because that token then already contains a "green light" after previous DCC validation during booking - so it would make things easier and faster when an additional scan of the DCC during entry is obsolete. But after thinking twice, I think it's out of scope for CWA, especially when tickets are printed or special apps from vendors (airlines) are used to store such a token.

File: Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <!-- XBUT: DCC Ticketing: Consent One: Cancel Dialog: Continue Button  -->
+    <string name="dcc_ticketing_consent_one_cancel_dialog_continue_btn">"Überprüfung Fortsetzen"</string>
+    <!-- XBUT: DCC Ticketing: Consent One: Cancel Dialog: Cancel Button  -->
+    <string name="dcc_ticketing_consent_one_cancel_dialog_cancel_btn">"Überprüfung Abbrechen"</string>
+
+    <!-- Not in Dcc ticketing allow list dialog-->
+    <!-- XTXT: not in AllowList dialog title -->
+    <string name="dcc_ticketing_not_in_allow_list_dialog_title">Sicherheitshinweis</string>
+    <!-- XTXT: not in AllowList dialog message -->
+    <string name="dcc_ticketing_not_in_allow_list_dialog_message">Der Anbieter ist uns nicht bekannt. Bitte überprüfen Sie, ob der QR-Code wirklich von dem Anbieter stammt und ob Sie ihm Ihr Zertifikat zur Verfügung stellen wollen.</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml:57]: This text was changed. It is now: "Der Anbieter ist uns nicht bekannt. Aus Sicherheitsgründen ist die Überprüfung bei Ticketbuchungen nur für bekannte Anbieter möglich. Bitte kontaktieren Sie den Anbieter, der Ihnen diesen QR-Code bereitgstellt hat."
See https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.11?node-id=37230%3A26423
Please adapt.

**Review by KathrinLuecke — COMMENTED** [review]
Please change dcc_ticketing_not_in_allow_list_dialog_message

**Pull request merged**